lib/game.js

```diff
@@ -19,7 +17,15 @@ var CanvasMotion = require('./canvas-motion');
 var canvasMotion = new CanvasMotion(canvasWidth, canvasHeight);
 var winsCounter = 0;
 
-canvas.addEventListener('keydown', function(key) {
+var $ = require('jquery');
+var p5Amplitude;
```

Comment:
want global scope, but want to define it within the loop because it changes instantaneously